.github/workflows/codecov.yml

```diff
@@ -8,13 +8,16 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
+      - uses: pnpm/action-setup@v4
+        with:
+          version: 9
```

Comment:
as I remember it will run `pnpm i` during this action execution. To disable that we need to add `run_install: false`. could you please check this out?